src/watts/plugin_sas.py

```diff
@@ -170,4 +184,8 @@ def postrun(self, params: Parameters, exec_info: ExecInfo) -> ResultsSAS:
             with open("PRIMAR4.dat", "r") as file_in, open("PRIMAR4.csv", "w") as file_out:
                 subprocess.run(str(self.conv_primar4), stdin=file_in, stdout=file_out)
 
+        if Path("MFUELss_C000001.dat").is_file(): # Timpano - May 2024: modifications to work with MFUEL output 
```

Comment:
remove comments `# Timpano - May 2024: modifications to work with MFUEL output `
This is tracked internally in GitHub

Reply:
Thank you. I confirm I removed the comments and committed the new changes.